components/base/Picturedbox.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from "@/styles/FeaturedWork.module.css";
+import { textFont, titleFont } from "../../pages/_app";
+
+interface MyProps {
+  location: string;
+  className: string;
+  title: string;
+  hasBorder: boolean;
```

Comment:
Do we really need this prop?

Reply:
Yes, I need these values to pass the value from FeaturedWork section to its child PicturedBox.

Reply:
Actually, I will delete className, while other props will reserved.